proxy/http2/Http2ConnectionState.cc

```diff
@@ -1100,33 +1125,6 @@ Http2ConnectionState::main_event_handler(int event, void *edata)
   }
   ++recursion;
   switch (event) {
-  // Initialize HTTP/2 Connection
-  case HTTP2_SESSION_EVENT_INIT: {
-    ink_assert(this->ua_session == nullptr);
-    this->ua_session = static_cast<Http2ClientSession *>(edata);
-    REMEMBER(event, this->recursion);
```

Comment:
We've been losing hints by removing calls for REMEMBER(). Isn't it worth keeping it? I'm fine with remove it temporally but want to put them back before we release 9.2.x (assuming we don't have this change on 9.1).

Reply:
Fair enough, I'll back REMEMBER.